tracks/dart/exercises/two-fer/mentoring.md

```diff
@@ -0,0 +1,56 @@
+### Reasonable solutions
+
+The most simple solution is to set the default value for the parameter to `"you"` and use string interpolation:
+
+```dart
+String twofer([String name = "you"]) {
+  return "One for $name, one for me.";
+}
+```
+
+As an alternative, one could use an expression-bodied method:
+
+```dart
+String twofer([String name = "you"]) => "One for $name, one for me.";
+```
+
+An alternative solution would be to use set the default value to `null` and use the 
+[null-coalescing operator][reference-null-coalescing-operator]:
+
+```dart
+String twofer([String name = null]) {
+  return "One for ${name ?? "you"}, one for me.";
+}
+```
+
+As an alternative, one could use an expression-bodied method here too:
+
+```csharp
+String twofer([String name = null]) => "One for ${name ?? "you"}, one for me.";
+```
+
+### Common suggestions
+
+- If they are using overloaded methods, suggest to use a single method with an 
+[optional parameter][reference-optional-parameter].
+- Instead of using an if-statement, suggest to use a single expression. This will force the student 
+to remove any duplication in the code.
+- Suggest using [string interpolation][reference-string-interpolation] over string concatenation 
+approach.
+- If they are using `null` as the default value, suggest to use `"you"` instead.
```

Comment:
If you give this advice, it should be correct in the solutions up top. Remove the one solution that uses `null` as the default value.